package/network/services/hostapd/patches/600-ubus_support.patch

```diff
@@ -458,3 +458,15 @@
  		case 'o':
  			params.override_driver = optarg;
  			break;
+--- a/src/ap/rrm.c
++++ b/src/ap/rrm.c
+@@ -89,6 +89,9 @@ static void hostapd_handle_beacon_report
+ 		return;
+ 	wpa_msg(hapd->msg_ctx, MSG_INFO, BEACON_RESP_RX MACSTR " %u %02x %s",
+ 		MAC2STR(addr), token, rep_mode, report);
++	if (len < sizeof(struct rrm_measurement_beacon_report))
++    	return;
```

Comment:
Indentation looks wrong

Reply:
I don't know why the Indentation looks wrong, but if you apply the patch, it is correct.

Reply:
You are mixing spaces and tabs, this is why it looks wrong. You are probably using 4 spaces for 1 tab, so it looks the same for you.

Reply:
@oldium whups, I'm sorry. I recreated the patch and now the indentation works.